trackpy/linking/linking.py

```diff
@@ -96,15 +92,23 @@ def link(f, search_range, pos_columns=None, t_column='frame', **kwargs):
         algorithm used to resolve subnetworks of nearby particles
         'auto' uses numba if available
         'drop' causes particles in subnetworks to go unlinked
+    neighbor_strategy : {'KDTree', 'BTree'}
+        algorithm used to identify nearby features. Default 'KDTree'.
+    to_eucl : function, optional
```

Comment:
Is this something that could be done by just processing the data before linking? As opposed to `dist_func`, which needs to be incorporated here. I am just thinking about our trajectory with data pre-processing in pims.